docs/rancher/csi-driver.md

```diff
@@ -121,6 +121,15 @@ Perform the following steps to deploy the Harvester CSI driver manually:
 
     ![](/img/v1.2/rancher/donot_change_cloud_config_path.png)
 
+:::note
+
+If you would prefer to install the Harvester CSI Driver without relying on the Rancher marketplace, you
+can do so with [Helm](https://helm.sh) since the Harvester CSI Driver [is packaged as a Helm chart](
+https://github.com/harvester/charts/tree/master/charts/harvester-csi-driver). See https://charts.harvesterhci.io
+for more information.
+
+:::
```

Comment:
That sounds good to me. Btw, I left it in the `::: note` block since that is more distinctive on the page but if you prefer the block quote `>` I can change it to that.

Reply:
@connorkuehl I follow whatever's in Markdown Guide (https://www.markdownguide.org/hacks/#admonitions) but I'm not sure if the guidelines are to be strictly followed. That said, we can keep what we have here and I'll just open a PR if I absolutely must change the markup.